cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOpenInvoiceData.ds

```diff
@@ -53,12 +53,12 @@ function getLineItems(args) {
 			|| (lineItem instanceof dw.order.PriceAdjustment && lineItem.promotion.promotionClass == dw.campaign.Promotion.PROMOTION_CLASS_ORDER)
 		) {
 			var lineItemObject = {};
-			var description = getDescription(lineItem),
-				id = getId(lineItem),
-				quantity = getQuantity(lineItem),
-				itemAmount: number = AdyenHelper.getCurrencyValueForApi(getItemAmount(lineItem)) / quantity,
-				vatAmount: number = AdyenHelper.getCurrencyValueForApi(getVatAmount(lineItem)) / quantity,
-				vatPercentage = getVatPercentage(lineItem);
+			var description = LineItemHelper.getDescription(lineItem),
+				id = LineItemHelper.getId(lineItem),
+				quantity = LineItemHelper.getQuantity(lineItem),
+				itemAmount = LineItemHelper.getItemAmount(lineItem) / quantity,
+				vatAmount = LineItemHelper.getVatAmount(lineItem) / quantity,
+				vatPercentage = LineItemHelper.getVatPercentage(lineItem);
```

Comment:
Be more defensive here. Add a `var` keyword per line, and add a `;` at the end of them.